SHTSensor.cpp

```diff
@@ -153,6 +153,82 @@ class SHTC1Sensor : public SHTI2cSensor
     }
 };
 
+//
+// class SHT2xSensor (SHT20, SHT21, SHT25)
+//
+
+class SHT2xSensor : public SHTI2cSensor
+{
+public:
+  SHT2xSensor(TwoWire &wire)
+      // clock stretching disabled
+      : SHTI2cSensor(0x40,     // i2cAddress
+                     0xF3F5,   // i2cCommand Hi: T, Lo: RH
+                     66,       // duration
```

Comment:
66 -> 85

66 is the typical duration, 85 the max

Reply:
Hm, ok. Will increase this. All my SHT21 (5 pieces) worked with 66 ... Done.